src/v2/api/index.md

```diff
@@ -1357,7 +1357,7 @@ type: api
 
 - **Usage:**
 
-  Watch an expression or a computed function on the Vue instance for changes. The callback gets called with the new value and the old value. The expression only accepts simple dot-delimited paths. For more complex expression, use a function instead.
```

Comment:
"simple" emphasizes the lack of complexity of the expression, rather than conceptual simplicity

Reply:
IMO it's superfluous, and reads more simply (ha!) without it.